File: src/main/java/com/ksptool/ql/biz/model/vo/RecoverRpChatVo.java
package com.ksptool.ql.biz.model.vo;

import lombok.Data;
import java.util.List;

@Data
public class RecoverRpChatVo {

    //角色对话ID
    private Long threadId;

    //AI模型代码
    private String modelCode;

    //对话串
    private List<RecoverRpChatHistoryVo> messages;

}

File: src/main/resources/views/aether/src/entity/RecoverRpChatVo.ts
import type RecoverRpChatHistoryVo from './RecoverRpChatHistoryVo';

/**
 * 恢复角色扮演聊天响应 VO
 */
export default interface RecoverRpChatVo {

    /**
     * 角色对话ID
     * Corresponds to Java Long
     */
    threadId: string; 

    /**
     * AI模型代码
     * Corresponds to Java String
     */
    modelCode: string;

    /**
     * 对话串
     * Corresponds to Java List<RecoverRpChatHistoryVo>
     */
    messages: RecoverRpChatHistoryVo[];

} 